src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -835,28 +836,83 @@ export default class Fms {
      * @param routeString {string}
      * @return {boolean}
      */
-    replaceDepartureProcedure(routeString) {
-        const routeStringElements = routeString.split(PROCEDURE_OR_AIRWAY_SEGMENT_DIVIDER);
+    replaceDepartureProcedure(routeString, airportIcao) {
+        const routeStringElements = routeString.toUpperCase().split(PROCEDURE_OR_AIRWAY_SEGMENT_DIVIDER);
```

Comment:
same thing here, please remove `toUpperCase()` (see previous comment)

Reply:
done

Reply:
@n8rzz actually, this one is needed. They are stored as uppercase in the routemodel.